src/BloomExe/web/EnhancedImageServer.cs

```diff
@@ -538,7 +538,8 @@ private bool ProcessCssFile(IRequestInfo info, string localPath)
 				if (_bloomBrowserUiCssFiles == null)
 				{
 					var sourceDir = FileLocator.GetDirectoryDistributedWithApplication("BloomBrowserUI");
-					_bloomBrowserUiCssFiles = Directory.EnumerateFiles(sourceDir, "*.css", SearchOption.AllDirectories).ToArray();
+					_bloomBrowserUiCssFiles = Directory.EnumerateFiles(sourceDir, "*.css", SearchOption.AllDirectories)
+						.Where(f => !f.Contains("nonProductionStuff")).ToArray();
```

Comment:
Feels to me as if we're getting excessively complex here. First we try the OptimizedFileLocator, which is initialized with many of the directories in BloomBrowserUI. If they aren't found we then search ALL those directories, including the ones we already tried, in an arbitrary (alphabetical?) order. And now it turns out some of them have to be explicitly suppressed!
Could we not tell the file locator positively about ALL the directories that need to be tried, and get rid of this extra step altogether?
The one thing to be careful of would be whether some other code is using the file locator and would be made to look in additional, undesirable places. But I doubt we want different parts of Bloom using different search paths for CSS files. That's a recipe for confusion...another reason NOT to have a separate, additional mechanism here.